### ISSUE TITLE
[BUG] keydown event is triggered only once if keyup flag is true

### ISSUE THREAD

**Issue by pandanoir** [JohannesKlauss/react-hotkeys-hook#889]
**Describe the bug**
If keyup and keydown flags are true, keydown event is triggered only once.

```
keydown is fired
keyup is fired
keyup is fired
keyup is fired
```

**To Reproduce**

```
useHotkeys('space', (event) => {console.log(event.type)}, {keydown: true, keyup: true});
```

**Expected behavior**

keydown event should be triggered.

**Comment by pandanoir** [comment]
https://github.com/JohannesKlauss/react-hotkeys-hook/blob/7d81556b7bea8fe6aa14e22ea8c61b0d25a61402/src/useHotkeys.ts#L106-L110

These lines should be swapped

```ts
 if (memoisedOptions?.keyup) { 
   listener(event) 
 } 

 hasTriggeredRef.current = false 
```

**Comment by pandanoir** [comment]
https://github.com/JohannesKlauss/react-hotkeys-hook/blob/7d81556b7bea8fe6aa14e22ea8c61b0d25a61402/src/useHotkeys.ts#L106-L110

`hasTriggeredRef.current = false` should be added

```ts
 hasTriggeredRef.current = false
 
 if (memoisedOptions?.keyup) {
   listener(event)
 }
 hasTriggeredRef.current = false
```

**Comment by JohannesKlauss** [comment]
Thanks for the issue and fix. Could you create a PR for this?

**Comment by pandanoir** [comment]
@JohannesKlauss 
I made a PR! Could you review the PR?

**Comment by JohannesKlauss** [comment]
Merged in 4.3.1
